src/components/widget/List/RawList.js

```diff
@@ -122,6 +121,35 @@ class RawList extends PureComponent {
     });
   }
 
+  handleOpenDropdown = () => {
+    const { handleClickOutside } = this;
+    const { onOpenDropdown, onFocus } = this.props;
+
+    onOpenDropdown();
+    onFocus();
```

Comment:
Order is important. We first focus the field, and then open the dropdown. This way it's easier to keep track of what's going on.